### PR TITLE
Form branch to development branch

### PR DESCRIPTION
**Pull request by daviidy**
**What we did**

- We built an HTML form that is almost identical to what we’d get by using a Rails helper so we could see how it’s done behind the scenes.
- We started morphing our form into a full Rails form with `#form_tag` and `form_for`

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @daviidy and @lucianosarno 
### You have done really well so far :clap: However, you have a few issues to fix:-
- [ ] Comment out the form_tag and use from_for as required from this project.
### Make the changes and submit for another code review. Good luck.
- Happy coding!

File: app/views/users/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+<%= form_tag("/users", method: "post") do %>
+  <%= label_tag(:username, "Username:") %>
+  <%= text_field_tag(:username) %>
+  <%= label_tag(:email, "Email:") %>
+  <%= email_field_tag(:email) %>
+  <%= label_tag(:password, "Password:") %>
+  <%= password_field_tag(:password) %>
+  <%= submit_tag("Search") %>
+<% end %>
```
Comment (rOluochKe) [inline app/views/users/edit.html.erb:23]: ### Cooment it out

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+<%= form_tag("/users", method: "post") do %>
+  <%= label_tag(:username, "Username:") %>
+  <%= text_field_tag(:username) %>
+  <%= label_tag(:email, "Email:") %>
+  <%= email_field_tag(:email) %>
+  <%= label_tag(:password, "Password:") %>
+  <%= password_field_tag(:password) %>
+  <%= submit_tag("Search") %>
+<% end %>
```
Comment (rOluochKe) [inline app/views/users/new.html.erb:22]: ### Cooment it out

File: app/views/users/edit.html.erb (hunk 1 above)
Comment (daviidy) [reply app/views/users/edit.html.erb:23]: But I commented with this block
```
<% if false %>
   ...
<% end %>
```

File: app/views/users/new.html.erb (hunk 2 above)
Comment (daviidy) [reply app/views/users/new.html.erb:22]: But I commented with this block
```
<% if false %>
   ...
<% end %>
```

**Review by rOluochKe — APPROVED** [review]
### You got it right, well done. The project has been approved. Congratulations :boom: :100:
### Merge and submit the project. Good luck with the next one.
- Happy coding!

**Pull request merged**